docs/source/quick_start/install.rst

```diff
@@ -0,0 +1,218 @@
+
+
+Using virtual environment
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For the most native, but more complicated option, you can create a virtual environment and install MCDR in it
+
+Create a virtual environment by:
+
+.. code-block:: bash
+
+    $ python3 -m venv <venv directory>
+
+``venv`` for example:
+
+.. code-block:: bash
+
+    $ python3 -m venv venv
+
+Activate it by:
+
++----------+------------+-----------------------------------------+
+| Platform | Shell      | Command to activate virtual environment |
++==========+============+=========================================+
+|  POSIX   | bash/zsh   | $ source venv/bin/activate              |
++          +------------+-----------------------------------------+
+|          | fish       | $ source venv/bin/activate.fish         |
++          +------------+-----------------------------------------+
+|          | csh/tcsh   | $ source venv/bin/activate.csh          |
++          +------------+-----------------------------------------+
+|          | PowerShell | $ venv/bin/Activate.ps1                 |
++----------+------------+-----------------------------------------+
+| Windows  | cmd.exe    | > venv\\Scripts\\activate.bat           |
++          +------------+-----------------------------------------+
+|          | PowerShell | PS > venv\\Scripts\\Activate.ps1        |
++----------+------------+-----------------------------------------+
+
+Then, install MCDR using pip:
+
+.. code-block:: bash
+
+    (venv) $ pip install mcdreforged
+
+When a new version of MCDR available, you may upgrade MCDR by:
+
+.. code-block:: bash
+
+    (venv) $ pip install mcdreforged -U
+
+An animated demo with bash:
+
+.. asciinema:: resources/venv.cast
+    :rows: 10
+
+|
+
+.. note::
+
+    In this way, you must activate the virtual environment every time you want to use MCDR, or install packages for MCDR plugins
+
+.. seealso ::
+
+    Python Doc: `How venvs work <https://docs.python.org/3/library/venv.html#how-venvs-work>`__
+
+Using Docker
+~~~~~~~~~~~~
+
+MCDR also provides Docker images as an option. See :doc:`/docker` for more details
+
+Specify MCDR version by specify the tag of Docker image. If you use ``latest``, MCDR should always being up to date
+
+Compared to the two methods above, Docker has a more convoluted learning path, but convenient for some advanced usages
+
+System package manager
+~~~~~~~~~~~~~~~~~~~~~~
+
+You may found MCDR in some package repositories, AUR for example. However, it's **definately not recommended** to use a system package manager to install MCDR. Not only it has all the same problem as the system-wide pip install, it's also hard to manage the dependencies of MCDR plugins
+
+Accelerate the installation
+---------------------------
+
+For users in some areas, ``pip`` and ``pipx`` may be very slow. You can use a mirror to accelerate the installation. Here's an example with 
```

Comment:
Just say `in China` instead of `in some areas`. You already provided a pypi mirror for China users

Also, "``pip`` and ``pipx`` may be very slow" is confusing. It's the network / download speed that is very slow